### PR TITLE
fixes deprecated APIs in wandb and keras

### PR DESCRIPTION
**Pull request by charlesfrye**
the accessing `validation_data` from `Callback.validation_data` [has been deprecated](https://github.com/tensorflow/tensorflow/issues/36375).

also, i believe we've changed how we log Media to runs, so `run.history.update` is no longer the move.

this PR will fix both of these issues, which strike the `Images` Callback in the autoencoder video.

### CODE REVIEW

**Comment by charlesfrye** [comment]
in the process, also detected and fixed a bug: the images weren't showing up in the media panel, but only for the convolutional autoencoder,.

turns out, that one had a `relu` activation in the output layer, instead of a softmax, meaning it could output values above 1, and image data is treated differently depending on its range.
